packages/v4/patternfly-docs/content/accessibility/progress/progress.md

```diff
@@ -0,0 +1,93 @@
+---
+id: Progress
+section: components
+---
+
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
+
+
+## Accessibility
+
+To implement an accessible PatternFly **progress** component:
+- Ensure a `title` or an `aria-label` are being used to give context to users. Additional context could also be provided via an `aria-labelledby` attribute mapping to the `id` of existing content on the page.
+- Ensure that a live region exists on the page through `aria-live` prior to updating the value of the progress component
+- Add a message to screen readers inside of the live region when the progress value updates, for example: <code class="ws-code">`Progress value is ${currentValue}%.`</code>
+
+
+## Testing
+
+At a minimum, a progress component should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="progress-a11y-checkbox-1" label={<span>The progress component has an accessible name via the <code class="ws-code">title</code> or <code class="ws-code">aria-label</code> attribute.</span>} description="This will give context when announced to users of assistive technologies when navigating through a page or a screen reader's rotor menu." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="progress-a11y-checkbox-2" label={<span>If progress status will dynamically appear or update, there is a live region with <code class="ws-code">aria-live="polite"</code> on page load where updates are rendered.</span>} description="This will allow assistive technologies to announce dynamically rendered updates." />
+  </ListItem>
+</List>
+
+
+## React customization
+
+Various React props have been provided for more fine-tuned control over accessibility.
+
+| Prop | Applied to | Reason | 
+|---|---|---|
+| `aria-label` | `Progress` | Adds accessible text to the ProgressBar. Required when title not used and there is not any label associated with the progress bar |
+| `aria-labelledby` | `Progress` | Associates the ProgressBar with it's label for accessibility purposes. Required when title not used |
+
```

Comment:
We could add in content for the `min`, `max`, `valueText`, and `label` props. `id` would be worth a mention since it can be used alongside the `title` prop for setting an aria-labelledby.

Reply:
I added them but not sure my description is perfect. 🙂